cmd/user-agent/src/pages/DIDManagement.vue

```diff
@@ -188,6 +188,10 @@ SPDX-License-Identifier: Apache-2.0
             const opts = await this.$trustblocStartupOpts
             this.didManager = new DIDManager(this.aries, this.$trustblocAgent, opts)
             this.searched = this.myData
+
+            // TODO to be replaced by actual user
+            this.username = this.$store.getters.getCurrentUser.username? this.$store.getters.getCurrentUser.username : 'demo-user'
```

Comment:
username should come from hub-auth's bootstrap data

Reply:
this us not wallet page, this is a demo page for creating DIDs.

Yes, once we start integrating hub-auth, it will be